platform-sdk/consensus-otter-tests/src/testFixtures/java/org/hiero/otter/fixtures/turtle/TurtleNodeConfiguration.java

```diff
@@ -20,22 +21,36 @@
  */
 public class TurtleNodeConfiguration extends AbstractNodeConfiguration {
 
+    /**
+     * Constructor for the {@link TurtleNodeConfiguration} class.
+     * <p>If this constructor is used, the {@link #setOutputDirectory(Path)} must be called before the node is started.
```

Comment:
Who will call `setOutputDirectory()`? It should not be called from tests, I assume.